packages/react/src/utils/loadClerkJsScript.ts

```diff
@@ -11,17 +11,31 @@ import { versionSelector } from './versionSelector';
 const FAILED_TO_LOAD_ERROR = 'Clerk: Failed to load Clerk';
 
 type LoadClerkJsScriptOptions = Omit<IsomorphicClerkOptions, 'proxyUrl' | 'domain'> & {
-  proxyUrl: string;
-  domain: string;
+  proxyUrl?: string;
+  domain?: string;
 };
 
-export const loadClerkJsScript = (opts: LoadClerkJsScriptOptions) => {
+const loadClerkJsScript = (opts: LoadClerkJsScriptOptions) => {
   const { publishableKey } = opts;
 
   if (!publishableKey) {
     errorThrower.throwMissingPublishableKeyError();
   }
 
+  const existingScript = document.querySelector<HTMLScriptElement>('script[data-clerk-js-script]');
+
+  if (existingScript) {
+    return new Promise((resolve, reject) => {
+      existingScript.addEventListener('load', () => {
+        resolve(existingScript);
+      });
+
+      existingScript.addEventListener('error', () => {
+        reject(FAILED_TO_LOAD_ERROR);
+      });
+    });
+  }
+
```

Comment:
This also allows anyone to add a script without waiting for an official support.